examples/text_summarization/bertsum_cnndm.ipynb

```diff
@@ -0,0 +1,3921 @@
+{
```

Comment:
<p>The description above sounds like we need to clone the entire BertSum repo here, but it looks like we are just copying a configuraiton file. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>